service-registry/registry-zero-config/src/main/java/org/apache/servicecomb/serviceregistry/client/ClientUtil.java

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Suggestions: package name can rename to `org.apache.serviecomb.zeroconfig`

Reply:
agree. will renmae package naeme